### PR TITLE
docs: Add test and linting steps to README [ggr]

### PR DESCRIPTION
**Pull request by miraleung**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -213,6 +213,26 @@ Contributions to this library are always welcome and highly encouraged.
...
+To run all tests:
+
+```sh
+$ bundle exec rake test
+```
+
+To test a single file:
+
+```sh
+$ bundle exec rake test TEST=path/to/test/file.rb
```
Comment (viacheslav-rostovtsev) [inline README.md:228]: This does not work from the root

**Review by viacheslav-rostovtsev — CHANGES_REQUESTED** [review]
If we are to add a developer guide, it needs to cover more ground, mentioning at least the generator structure and describing the development workflow, e.g. protos vs bin, unit tests vs gold tests, vs unit-tests-on-gold vs showcase. 

As it is, it's only useful to a very narrow scope of potential developers, not to mention being wrong, e.g. 
`bundle exec rake test TEST=path/to/test/file.rb` does not do what this says when run from the root of the project, and the distinction of root vs subgenerator is not described in the guide.

File: README.md (hunk 1 above)
Comment (miraleung) [reply README.md:239]: Fixed.

**Comment by miraleung** [comment]
This PR is meant to incrementally add developer steps that would aid in setting up and working with this codebase. A full developer guide could come as a result of similar accumulated incremental changes, or this PR could make such an effort easier.

**Review by viacheslav-rostovtsev — APPROVED** [review]

**Pull request merged**